sdf.md

```diff
@@ -173,7 +173,9 @@ document that a specific extension is in effect; see
 {{information-block}} for details of the `feature` quality that can be
 used for such indications.
 Base SDF does not define a "version" concept for the SDF specification
-itself (as opposed to SDF documents); however there may be occasional
+itself (as opposed to version indications for the specifications
```

Comment:
Is “specifications in SDF documents” terminology we use elsewhere? (On mobile now, difficult to check)

Reply:
See below ("Improve...").